tests/test_processes/wps_test_collect_args.py

```diff
@@ -0,0 +1,58 @@
+from pywps import Process, LiteralInput, ComplexInput, LiteralOutput, FORMATS
+from wps_tools.utils import collect_args
+import logging
+
+
+class TestCollectArgs(Process):
```

Comment:
We may want to consider turning these test processes into pytest fixtures in a `conftest.py` module. That was we can pass them as a parameter rather than import them.